frontend/src/metabase/admin/settings/components/widgets/SettingToggle.jsx

```diff
@@ -4,13 +4,18 @@ import { t } from "ttag";
 import Toggle from "metabase/core/components/Toggle";
 import Tooltip from "metabase/core/components/Tooltip";
 
-const SettingToggle = ({ setting, onChange, disabled, tooltip }) => {
+const SettingToggle = ({ id, setting, onChange, disabled, tooltip }) => {
   const value = setting.value == null ? setting.default : setting.value;
   const on = value === true || value === "true";
   return (
     <div className="flex align-center pt1">
       <Tooltip tooltip={tooltip} isEnabled={!!tooltip}>
-        <Toggle value={on} onChange={!disabled ? () => onChange(!on) : null} />
+        <Toggle
+          id={id}
+          value={on}
+          onChange={!disabled ? () => onChange(!on) : null}
+          disabled={disabled}
```

Comment:
Do we need to check for `disabled` when assigning `onChange`? I believe `disabled` should disable clicks anyway?